### PR TITLE
Update index.md

### PR DESCRIPTION
**Pull request by holmesb**
Fixes issue #3

### CODE REVIEW

**Review by hbokh — APPROVED** [review]
LGTM

**Comment by hbokh** [comment]
@SchoolGuy I am unable to merge, since "Only those with write access to this repository can merge pull requests."

File: docs/index.md [hunk 1]
```diff
@@ -13,6 +13,15 @@ to be configured with the proper credentials before it can be used.
 
 Use the navigation to the left to read about the available resources.
 
+## Prereq
+
+Until [this issue](https://github.com/cobbler/cobbler/issues/2386) is 
+fixed, it is necessary to disable Cobbler's cache by adding the following
+line to /etc/cobbler/settings file:
```
Comment (SchoolGuy) [inline docs/index.md:20]:
```suggestion
line to `/etc/cobbler/settings` (since Cobbler 3.2.1 `settings.yaml` file:
```

**Review by SchoolGuy — CHANGES_REQUESTED** [review]
There was a change in 3.2.1 which needs to be thought of.

File: docs/index.md (hunk 1 above)
Comment (holmesb) [reply docs/index.md:20]: Missing bracket?

Comment (SchoolGuy) [reply docs/index.md:20]: Yes you are right. Fixed it.

**Review by SchoolGuy — APPROVED** [review]
LGTM

**Pull request merged**